app/views/checkout/payment.scala.html

```diff
@@ -83,23 +83,23 @@
                             <div class="form-field js-checkout-house">
                                 <label class="label" for="address-line-1">Address line 1</label>
                                 <input type="text" class="input-text js-input" id="address-line-1" name="personal.address.address1" value="@form.data.get("personal.address.address1")">
-                                @fragments.forms.errorMessage("This field is required")
+                                @fragments.forms.errorMessage("")
                             </div>
                             <div class="form-field js-checkout-street">
                                 <label class="label optional-marker" for="address-line-2">Address line 2</label>
                                 <input type="text" class="input-text js-input" id="address-line-2" name="personal.address.address2" value="@form.data.get("personal.address.address2")">
-                                @fragments.forms.errorMessage("This field is required")
+                                @fragments.forms.errorMessage("")
```

Comment:
Is it worth still providing a default message, even if we override in most cases?